addons/account/tests/test_reconciliation.py

```diff
@@ -1913,12 +1913,15 @@ def test_reconciliation_cash_basis_fx_02(self):
     def test_reconciliation_cash_basis_revert(self):
         company = self.env.ref('base.main_company')
         company.tax_cash_basis_journal_id = self.cash_basis_journal
-        tax_cash_basis10percent = self.tax_cash_basis.copy({'amount': 10})
-        self.tax_waiting_account.reconcile = True
         tax_waiting_account10 = self.tax_waiting_account.copy({
             'name': 'TAX WAIT 10',
             'code': 'TWAIT1',
         })
+        tax_cash_basis10percent = self.tax_cash_basis.copy({
+            'amount': 10,
+            'cash_basis_transition_account_id': tax_waiting_account10.id,
+        })
+        self.tax_waiting_account.reconcile = True
```

Comment:
Why this change ?

Reply:
This is now considered as a refund, therefore revers lines have their account updated by _get_default_tax_account. The tax cash_basis_transition_account_id is now used instead of the original line account id.
As 'tax_cash_basis10percent' is a copy of 'tax_cash_basis' its needs it's cash_basis_transition_account_id updated. If not the reversed tax line wil get the tax_cash_basis cash_basis_transition_account_id instaed of  tax_waiting_account10 and reconciliation will not be possible.

Reply:
This is representing a bug in 13.0... the transition tax account is lost when reverting...